main.tex

```diff
@@ -580,16 +579,16 @@ \subsubsection{Local Earth coordinate frames}
 
 \subsubsection{Proper motion and velocity transformations}
     In addition to positional coordinate data, the \texttt{Frame} classes now
-    also support velocity data.
+    also support velocity.
     As the default representation for most frames is spherical, most of the
     \texttt{Frame} classes expect proper motion and radial velocity components
     to specify the velocity information.
     The names of the proper motion components all start with \texttt{pm} and
     adopt the same longitude and latitude names as the positional components.
     Transforming coordinates with velocity data is also supported, but in some
     cases the transformed velocity components have limited accuracy because the
-    transformations are done numerically.
-    The visualization of the coordinate frame transform graph highlights which
+    transformations are done numerically instead of analytically.
```

Comment:
@eteq or @adrn might need confirm whether changes here are correct. My edits are intended to clarify things for readers.